db/trie/mptrie/merklepatriciatrie_benchmark_test.go

```diff
@@ -92,3 +93,83 @@ func benchTrieUpsert(b *testing.B, e binary.ByteOrder) {
 	b.StopTimer()
 	b.ReportAllocs()
 }
+
+func BenchmarkTrie_UpertWithAsync(b *testing.B) {
+	tr, _, callback := initTrie(2)
+	defer callback()
+
+	keys := [][]byte{}
+	for i := 0; i < 10; i++ {
+		for j := 0; j < 256; j++ {
+			key := []byte{byte(i), byte(j)}
+			keys = append(keys, key)
+		}
+	}
```

Comment:
with keyLen = 2, and first byte = 0~9, would it be a relatively simple scenario that may not reflect actual use-case?
perhaps use keyLen = 4, and key = random(0, 2^32-1)?

Reply:
Good Suggestion. But I think randomized trie structure might influence benchmark performance